pkg/api/customization/node/node_driver.go

```diff
@@ -43,6 +43,17 @@ func (h *DriverHandlers) ActionHandler(actionName string, action *types.Action,
 	case "deactivate":
 		m.Spec.Active = false
 		v3.NodeDriverConditionInactive.Unknown(m)
+		var newConditions []v3.Condition
```

Comment:
@kinarashah can we put a comment in the code explaining why we remove these conditions on deactivate